modules/qtwidgets/src/properties/stringmultilinepropertywidgetqt.cpp

```diff
@@ -104,6 +105,59 @@ void StringMultilinePropertyWidgetQt::updateFromProperty() {
     }
 }
 
+MultiFileStringPropertyWidgetQt::MultiFileStringPropertyWidgetQt(MultiFileProperty* property)
+    : PropertyWidgetQt(property), property_(property) {
+
+    QHBoxLayout* hLayout = new QHBoxLayout;
+    setSpacingAndMargins(hLayout);
+
+    label_ = new EditableLabelQt(this, property_);
+    hLayout->addWidget(label_);
+
+    textEdit_ = new MultilineTextEdit;
```

Comment:
The constructor should be called.
```suggestion
    textEdit_ = new MultilineTextEdit();
```

Reply:
the initializer is optional here https://en.cppreference.com/w/cpp/language/new

Reply:
But sure we usually put one in